configure.ac

```diff
@@ -42,6 +42,16 @@ debug_c_warn_flags="-Wextra -Wno-unused-parameter -Wno-sign-compare -Wno-missing
 debug_c_other_flags="-fstack-protector-strong"
 picky_c_warn_flags="-Wno-long-long -Wmissing-prototypes -Wstrict-prototypes -Wcomment -pedantic"
 
+AC_ARG_WITH([version_suffix],
+	    [AC_HELP_STRING([--with-version_suffix],
+			    [Enable version-suffix annotations @<:@default=no@:>@])],
+	    [], [with_version_suffix=no])
+AS_IF([test x"$with_version_suffix" != x"no"],
```

Comment:
What if I want to have "no" as the suffix? For "--with" options, we can take empty value as default.

Reply:
![image](https://user-images.githubusercontent.com/617316/36268260-08c57aae-1244-11e8-9922-e83c33aa3351.png)

Reply:
Joke gif aside, `--without-foo` gives you a shell variable of `$with_foo` equal to `no`.  My $0.02 is that this is an acceptable limitation; I don't think we'll realistically ever want to have a version suffix of `no`.